test/string.test.ts

```diff
@@ -71,6 +77,14 @@ describe("String", () => {
                 "com",
             ])
         })
+
+        it("handles the separator not existing in the string", () => {
```

Comment:
So this didn't actually increase coverage at all but felt like a worthwhile test case to me. Happy to remove if you disagree.